saleor/graphql/warehouse/schema.py

```diff
@@ -46,8 +47,8 @@ class WarehouseQueries(graphene.ObjectType):
     @traced_resolver
     def resolve_warehouse(self, info, **data):
         warehouse_pk = data.get("id")
-        warehouse = graphene.Node.get_node_from_global_id(info, warehouse_pk, Warehouse)
-        return warehouse
+        _, id = from_global_id_or_error(warehouse_pk, Warehouse)
+        return models.Warehouse.objects.filter(id=id).first()
```

Comment:
Maybe we should extract ORM to dedicated resolvers file like we've done in many other places?